React/Views/RCTFont.mm

```diff
@@ -33,6 +33,19 @@
 
 #endif
 
+static NSArray *fontNameOrdering = [NSArray arrayWithObjects:
```

Comment:
Let's initialize this array inside the dispatch once block, just like the `nameToWeight` dictionary.

Reply:
Use `@[a, b, c]` shorthand to initialize the array instead of `arrayWithObjects`.